oak/server/oak_node.cc

```diff
@@ -209,7 +209,7 @@ void OakNode::Start() {
   // we pass the DefinedNode* to the thread by value so it doesn't fall out
   // of scope once this function exits.
   LOG(INFO) << "Executing module oak_main on new thread";
-  std::thread t([this] { this->RunModule(); });
+  std::thread t(&OakNode::RunModule, this);
   main_ = std::move(t);
```

Comment:
Even more idiomatic...

oak_main_thread_ = std::thread(&OakNode::RunModule, this);

Reply:
Done.